clang/test/compiler-course/beskhmelnova_k_implicit_conversions/beskhmelnova_k_implicit_conversions_test.cpp

```diff
@@ -0,0 +1,32 @@
+// RUN: %clang_cc1 -load %llvmshlibdir/ClangAST_1_BeskhmelnovaK_FIIT1_ClangAST%pluginext -plugin ClangAST_1_BeskhmelnovaK_FIIT1_ClangAST -fsyntax-only %s 2>&1 | FileCheck --match-full-lines %s
+
+// Проверка порядка выводимых строк:
+// CHECK: Function sum
+// CHECK-NEXT: float -> double: 1
+// CHECK-NEXT: int -> float: 1
```

Comment:
It is better to insert checks immediately before the tested code
```suggestion
// CHECK: Function sum
// CHECK-NEXT: float -> double: 1
// CHECK-NEXT: int -> float: 1
double sum(int a, float b) {
    return a + b;
}
```